scripts/tmuxinator.sh

```diff
@@ -18,13 +18,9 @@ export FM_VERBOSE_OUTPUT=0
 source scripts/build.sh
 echo "Running in temporary directory $FM_TEST_DIR"
 
-env | sed -En 's/(FM_[^=]*).*/\1/gp' | while read var; do printf 'export %s=%q\n' "$var" "${!var}"; done > .tmpenv
-
 export FM_BITCOIND_RPC="http://bitcoin:bitcoin@127.0.0.1:18443"
```

Comment:
if I compare the `.tmpenv` file with the env variables set in the tmux session (`env | rg '^FM_.*'`) then there is also this `FM_BITCOIND_RPC` variable (which is not in .tmpenv).